packages/gatsby-theme/src/components/Share/ShareSocial.js

```diff
@@ -0,0 +1,79 @@
+import { forwardRef } from "react"
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton,
+  RedditShareButton,
+  WhatsappShareButton,
+} from "react-share"
+import Icon from "../Icon"
+
+const Socials = {
+  facebook: FacebookShareButton,
+  linkedin: LinkedinShareButton,
+  twitter: TwitterShareButton,
+  reddit: RedditShareButton,
+  whatsapp: WhatsappShareButton,
+}
+
+const ShareSocial = forwardRef(
+  (
+    {
+      social,
+      title,
+      to,
+      handler,
+      hashtags,
+      children,
+      summary,
+      source,
+      separator,
+      ...props
+    },
+    ref
+  ) => {
+    return (
+      <Box
+        summary={social === "linkedin" && summary}
+        source={source === "linkedin" && source}
+        separator={social === "whatsapp" && separator}
+        via={social === "twitter" && handler}
+        hashtags={social === "twitter" && hashtags}
```

Comment:
I think we can update this part to remove those strings
```suggestion
        summary={summary}
        source={source}
        separator={separator}
        via={handler}
        hashtags={hashtags}
```